lib/glific/searches/full.ex

```diff
@@ -38,6 +40,23 @@ defmodule Glific.Search.Full do
 
   defp run_include_groups(query, _args), do: query
 
+  @spec run_include_labels(Ecto.Queryable.t(), map()) :: Ecto.Queryable.t()
+  defp run_include_labels(query, label_ids) when is_list(label_ids) and label_ids != [] do
+    flow_labels =
+      FlowLabel
+      |> where([f], f.id in ^label_ids)
+      |> select([f], f.name)
+      |> Repo.all()
+
+    flow_labels
+    |> Enum.reduce(query, fn flow_label, query ->
+      where(query, [m: m], ilike(m.flow_label, ^"%#{flow_label}%"))
+    end)
+    |> or_where([m: m], ilike(m.flow_label, ^"%#{flow_labels}%"))
+  end
+
```

Comment:
isn't this easier:

query = query |> where([m: m], m.flow_label in ^flow_labels)

Reply:
@dlobo I have update the query. Thanks